### ISSUE TITLE
一点小问题

### ISSUE THREAD

**Issue by wlhgtc** [liyucheng09/llm-compressive#1]
从知乎看了这个工作感觉很有趣，想看下代码实现，碰到了些问题：
[这里](https://github.com/liyucheng09/llm-compressive/blob/8b9b5b8aa8462f3756ac3239b8f7f16d32cf3531/compressor.py#L14) 的 `logits` 我理解应该是 `[batch, seq_length, vocab_size]`，
不太理解为什么在 `vocab_size`维度做 `cumsum`, 是为了后面计算 `P(X)` 吗?
因为看不懂 C 的代码，只能理解到[这里](https://github.com/liyucheng09/llm-compressive/blob/8b9b5b8aa8462f3756ac3239b8f7f16d32cf3531/arithmetic.py#L85) 是在做论文中的 **Algorithm 1 Arithmetic Coding**

**Comment by wlhgtc** [comment]
@liyucheng09

**Comment by liyucheng09** [comment]
了解，我这两天再写篇文章，解释一下用LLMs做压缩的流程。在里边会介绍一下代码实现

**Comment by wlhgtc** [comment]
> 了解，我这两天再写篇文章，解释一下用 LLMs 做压缩的流程。在里边会介绍一下代码实现

突然想起来了：这个累加的区间应该就是 Arithmetic Coding 要的那个概率区间
